src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -79,13 +79,22 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
     return index;
   }, [activeIndex, labels.length]);
 
+  const tablistRef = React.useRef<HTMLUListElement>(null);
+
   const [currentIndex, setCurrentIndex] = React.useState(getValidIndex());
+  const [stripeStyle, setStripeStyle] = React.useState<React.CSSProperties>({});
 
   React.useEffect(() => {
     if (activeIndex != null && currentIndex !== activeIndex) {
       setCurrentIndex(getValidIndex());
     }
-  }, [activeIndex, currentIndex, getValidIndex]);
+    const activeTab = tablistRef.current?.children[currentIndex] as HTMLElement;
+    type !== 'default' &&
+      setStripeStyle({
```

Comment:
We need to handle resizes before release.

Reply:
But do we really?

Reply:
Maybe I need to reopen #75 😄

Reply:
I can add it in a separate PR. Need to merge this one as it's blocking things